### PR TITLE
SDK:  Speed up tests

### PR DESCRIPTION
**Pull request by xitronix**

### CODE REVIEW

**Comment by marekkirejczyk** [comment]
Let's make consistent naming (step, tick => tick)
Let's introduce (
```ts
SdkConfig. authorisationsObserverTick
SdkConfig.executionFactoryTick
```

File: universal-login-sdk/lib/config/SdkConfig.ts [hunk 1]
```diff
@@ -4,4 +4,6 @@ export interface SdkConfig {
   paymentOptions: PaymentOptions;
   observedTokensAddresses: string[];
   observedCurrencies: ObservedCurrency[];
+  authorisationsObserverTick: number;
```
Comment (marekkirejczyk) [inline universal-login-sdk/lib/config/SdkConfig.ts]: `authorisationsObserver: {tick: number, timeout: number}`

File: universal-login-sdk/test/integration/core/AuthorisationObserver.ts [hunk 2]
```diff
@@ -38,7 +38,7 @@ describe('INT: AuthorisationsObserver', async () => {
     ({sdk, relayer, contractAddress, privateKey, wallet} = await loadFixture(basicSDK));
     getAuthorisationRequest = createGetAuthorisationRequest(contractAddress, privateKey);
     ({authorisationsObserver} = sdk);
-    authorisationsObserver.step = 50;
+    authorisationsObserver.tick = 50;
```
Comment (marekkirejczyk) [inline universal-login-sdk/test/integration/core/AuthorisationObserver.ts]: unnecessary

File: universal-login-sdk/test/integration/core/BlockchainObserver.js [hunk 3]
```diff
@@ -27,7 +27,7 @@ describe('INT: BlockchainObserver', async () => {
   beforeEach(async () => {
     ({relayer, sdk, mockToken, contractAddress, wallet, otherWallet, otherWallet2, privateKey} = await loadFixture(basicSDK));
     ({blockchainObserver} = sdk);
-    blockchainObserver.step = 50;
+    blockchainObserver.tick = 50;
```
Comment (marekkirejczyk) [inline universal-login-sdk/test/integration/core/BlockchainObserver.js]: move to contructor

File: universal-login-sdk/test/integration/core/BalanceObserver.ts [hunk 4]
```diff
@@ -26,7 +26,7 @@ describe('INT: BalanceObserver', () => {
 
       balanceChecker = new BalanceChecker(provider);
       balanceObserver = new BalanceObserver(balanceChecker, TEST_ACCOUNT_ADDRESS, {tokensDetails: supportedTokens} as TokensDetailsStore);
-      balanceObserver.step = 100;
+      balanceObserver.tick = 100;
```
Comment (marekkirejczyk) [inline universal-login-sdk/test/integration/core/BalanceObserver.ts]: move to constructor

File: universal-login-wallet/test/integration/services/LoginServices.ts [hunk 5]
```diff
@@ -35,7 +35,7 @@ describe('Login', () => {
     walletServiceForConnect = new WalletService(sdk);
     connectToWalletService = ConnectionToWalletService(sdk, walletServiceForConnect);
     ({blockchainObserver} = sdk);
-    blockchainObserver.step = 10;
+    blockchainObserver.tick = 10;
```
Comment (marekkirejczyk) [inline universal-login-wallet/test/integration/services/LoginServices.ts]: unnecessary

**Comment by marekkirejczyk** [comment]
Next time:

type ObserverOptions {
  tick: number
  timeout: number
}

File: universal-login-sdk/lib/config/SdkConfig.ts (hunk 1 above)
Comment (JustynaBroniszewska) [reply universal-login-sdk/lib/config/SdkConfig.ts]: ` authorisationsObserverTick: number` and `executionFactoryTick: number` should be optional

**Pull request merged**